inc/cleantalk-settings.php

```diff
@@ -3629,3 +3649,52 @@ function apbct_settings_field__action_adjust()
     $res = AdjustToEnvironmentSettings::render();
     echo $res;
 }
+
+function apbct_settings_get_form_decorator_select($params, $disabled, $value)
```

Comment:
So many duplicated code here. Is it possible to rewrite te logic without code duplication? Maybe you can prepare all data in the `options_callback` attribute?